Develop doesn't build with GNU

Per last night:
```
/discover/nobackup/mathomp4/SystemTests/builds/AGCM_GITHUBDEVGNU/CURRENT/GEOSgcm/src/Shared/@MAPL/MAPL_Base/MAPL_ExtDataGridCompMod.F90:2982:150:

                  Write(*,'(a,4(L1,x),a,a)') ' DEBUG: Extrapolation flags (2) are ',LExact,RExact,LExtrap,RExtrap,'for file ', trim(fdata%get_file_name)
                                                                                                                                                      1
Error: Expected argument list at (1)
make[2]: *** [src/Shared/@MAPL/MAPL_Base/CMakeFiles/MAPL_Base.dir/MAPL_ExtDataGridCompMod.F90.o] Error 1
make[1]: *** [src/Shared/@MAPL/MAPL_Base/CMakeFiles/MAPL_Base.dir/all] Error 2
make: *** [all] Error 2
```

--- a/MAPL_Base/MAPL_ExtDataGridCompMod.F90
+++ b/MAPL_Base/MAPL_ExtDataGridCompMod.F90
@@ -2979,7 +2979,7 @@ CONTAINS
               LExact  =  (cLimTime == tSeries(1))
               RExact  =  (cLimTime == tSeries(nsteps))
               If (Mapl_Am_I_Root().and.(Ext_Debug > 19)) Then
-                 Write(*,'(a,4(L1,x),a,a)') ' DEBUG: Extrapolation flags (2) are ',LExact,RExact,LExtrap,RExtrap,'for file ', trim(fdata%get_file_name)
+                 Write(*,'(a,4(L1,x),a,a)') ' DEBUG: Extrapolation flags (2) are ',LExact,RExact,LExtrap,RExtrap,'for file ', trim(fdata%get_file_name())
               End If
 
            End IF

